mapie/regression.py

```diff
@@ -608,71 +597,74 @@ def predict(
         # Checks
         check_is_fitted(self, self.fit_attributes)
         self._check_ensemble(ensemble)
-        alpha_ = check_alpha(alpha)
+        alpha = cast(Optional[NDArray], check_alpha(alpha))
         y_pred = self.single_estimator_.predict(X)
+        n = len(self.conformity_scores_)
 
         if alpha is None:
             return np.array(y_pred)
+
+        alpha_np = cast(NDArray, alpha)
```

Comment:
`alpha` was not converted to `NDArray` in line 600 ?

Reply:
No, to `Optional[NDArray]`, so that checking against `None` is valid for mypy